x-pack/plugins/features/server/oss_features.ts

```diff
@@ -156,6 +183,7 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
               'lens',
               'map',
               'tag',
+              'background-session',
```

Comment:
We don't need to specify this here since it's being granted by the sub-feature privilege below:
```suggestion
```